main.go

```diff
@@ -41,29 +45,38 @@ func main() {
 	discord.AddHandler(handleVoiceStateUpdate)
 	// Keep the bot running
 	log.Println("Bot is now running. Press Ctrl+C to exit.")
+	birthdateCheck(discord)
+	timer := time.NewTicker(24 * time.Hour)
+	go func() {
+		for range timer.C {
+			now = time.Now()
+			birthdateCheck(discord)
+		}
+
+	}()
```

Comment:
Should move this into a function (say `birthdateCheckRoutine`) and then you can `go birthdateCheckRoutine()` here, and handle the initial run / subsequent scheduling within the function.